src/lib/notificationWorker.js

```diff
@@ -15,6 +15,7 @@ class NotificationWorker {
     this.Transfer = Transfer
     this.Subscription = Subscription
     this.Fulfillment = Fulfillment
+    this.config = require('../services/config')
```

Comment:
@MatthewPhinney good point, I got confused with connector. I updated the comment